ZenPacks/zenoss/OpenStackInfrastructure/datasources/HeartbeatsAMQPDataSource.py

```diff
@@ -0,0 +1,88 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2014-2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger('zen.OpenStack.HeartbeatsAMQP')
+
+from twisted.internet.defer import inlineCallbacks, succeed
+
+from zope.component import adapts
+from zope.interface import implements
+
+from ZenPacks.zenoss.OpenStackInfrastructure.datasources.AMQPDataSource import (
+    AMQPDataSource, AMQPDataSourcePlugin, AMQPDataSourceInfo,
+    IAMQPDataSourceInfo)
+
+
+class HeartbeatsAMQPDataSource(AMQPDataSource):
+    '''
+    Datasource used to capture heartbeats shipped to us from OpenStack
```

Comment:
probably just update this comment to say that it is deprecated and no longer does anything